mbedtls/tests/save_restore.rs

```diff
@@ -46,6 +48,7 @@ fn save_restore_aes_cbc_enc_nopad() {
     assert_eq!(&ct[0..32], &expected_ct[..]);
 }
 
+#[cfg(buggy)]
```

Comment:
I guess this is disabling the test?

It's more normal to use `#[ignore]` can we do that instead?
Reading this code sent me on a doc search adventure trying to figure out if buggy has some special meaning